pkgs/top-level/all-packages.nix

```diff
@@ -5980,6 +5980,16 @@ with pkgs;
     openssl = null;
   };
 
+  openconnect-globalprotect = openconnect-globalprotect_gnutls;
```

Comment:
I think this attribute be enough for the top level, and leave the rest of the configuration to overrides. Three versions seems a bit much.